c7n/tags.py

```diff
@@ -692,134 +694,6 @@ def get_client(self):
                 self.manager.resource_type.service)
 
 
-class NormalizeTag(Action):
```

Comment:
nit, moving the class in the file caused a more significant diff then what actually took place.

Reply:
Good call - this PR may or may not be useful based on other discussions, but as a matter of general PR hygiene I split the move into a separate commit so only the more relevant changes show up [here](https://github.com/cloud-custodian/cloud-custodian/pull/6837/commits/6eba6d33b4a2b25e11e8873f8578bf4013a8a50a).